webcompat/webhooks/model.py

```diff
@@ -185,12 +189,16 @@ def close_public_issue(self, reason='rejected'):
         Right now the accepted reasons are:
         'incomplete'
         'invalid'
+        'autoclosed'
         'rejected' (default)
         """
         if reason == 'incomplete':
             payload_request = self.prepare_accepted_issue('incomplete')
         elif reason == 'invalid':
             payload_request = self.prepare_accepted_issue('invalid')
+        elif reason == 'autoclosed':
+            payload_request = prepare_rejected_issue(reason)
+            payload_request['body'] += prepare_private_url(self.html_url)
```

Comment:
I have added a hidden private issue github url to the closed public issue because I need to use it to train bugbug model. Since there will be no way to get the original content of a public issue, the plan is to make a request to fetch data of a private issue and use its contents instead, as it will be useful for training the model.

So it looks like this:

```
<p>Thanks for the report. We have closed this issue 
automatically as we suspect it is invalid. If we made a mistake, please 
file a new issue and try to provide more context or file a report 
<a href="https://github.com/webcompat/web-bugs/issues/new/choose">
via our GitHub repository</a>.</p>
<!-- @private_url: repos/webcompat/webcompat-tests-private/issues/273 -->
```

Reply:
Maybe we could add a comment explaining just that. :)

Reply:
done :)